tealeaf/TeaLeafIOS.entitlements

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-<dict/>
+<dict>
+	<key>com.apple.security.application-groups</key>
+	<array>
+		<string></string>
+	</array>
+</dict>
```

Comment:
this will be later moved to module, there build.js auto handles merging of entitlement files. that's why I did like this.